src/rmodels.c

```diff
@@ -508,6 +508,8 @@ void DrawCylinder(Vector3 position, float radiusTop, float radiusBottom, float h
 {
     if (sides < 3) sides = 3;
 
+    const float angle_step = 360.0f / sides;
```

Comment:
Variable name and spacing is not following [raylib code naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md).